cmd/qat_plugin/README.md

```diff
@@ -276,19 +276,39 @@ $ kubectl get pods
 > **Note**: The deployment example above uses [kustomize](https://github.com/kubernetes-sigs/kustomize)
 > that is available in kubectl since Kubernetes v1.14 release.
 
-#### Run the test
+#### Manual test run
```

Comment:
You need to update the TOC. And add a new entry as well I think.
Not too hard to do manually, but I've been using the [kata check-markdown tool](https://github.com/kata-containers/tests/tree/master/cmd/check-markdown) to do it:

```bash
$ check-markdown toc README.md
```

Reply:
updated